db/migrate/20180219151742_add_metadata_fields_to_dimensions_item.rb

```diff
@@ -0,0 +1,9 @@
+class AddMetadataFieldsToDimensionsItem < ActiveRecord::Migration[5.1]
+  def change
+    add_column :dimensions_items, :document_type, :string
+    add_column :dimensions_items, :content_purpose_supertype, :string
+    add_column :dimensions_items, :first_published_at, :datetime
+    add_column :dimensions_items, :public_updated_at, :datetime
+    add_column :dimensions_items, :last_updated_time, :datetime
```

Comment:
It might be better to follow the same convention of using the suffix: `at`. I would rename it to `last_updated_at`?